README.md

```diff
@@ -1,94 +1,60 @@
 # OK
 
-**Elegant error handling in elixir pipelines. See [Handling Errors in Elixir](http://insights.workshop14.io/2015/10/18/handling-errors-in-elixir-no-one-say-monad.html) for a more detailed explanation**
+**Elegant error handling for Elixir. Built on the solid foundation of the result monad.**
 
-[Documentation for OK is available on hexdoc](https://hexdocs.pm/ok)
-
-## Installation
```

Comment:
Thought i'd remove this whole section as elixir users know what to do and it says the same on hex anyway